packages/schema/lib/schemas/LockObjectSchema.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+const makeSchema = require('../utils/makeSchema');
+
+module.exports = makeSchema({
+  id: '/LockObjectSchema',
+  description:
+    '**INTERNAL USE ONLY**. Zapier uses this config for internal operation locking.',
+  type: 'object',
+  required: ['key'],
+  properties: {
+    key: {
+      description:
+        'The key to use for locking. This should be unique to the operation. While actions of different integrations with the same key and scope will never lock each other out, actions of the same integration with the same key and scope will do. User data provided for the input fields can be used in the key with the use of the curly braces referencing. For example, to access the user data provided for the input field "test_field", use `{{bundle.inputData.test_field}}`. Note that a required input field should be referenced to get user data always.',
```

Comment:
@FokkeZB The descriptions for the lock object fields have been updated to cater to your feedback. Please, review it again. Thanks.